dashboard/app/views/levels/_teacher_panel.html.haml

```diff
@@ -17,9 +17,10 @@
     - level_example_links = @level.examples.map do |example|
       // We treat Sprite Lab levels as a sub-set of game lab levels right now which breaks their examples solutions
       // as level.game.app gets "gamelab" which makes the examples for sprite lab try to open in game lab.
-      // We treat Dancelab as a sub-set of Sprite Lab levels so have to check and set that before GamelabJr
+      // We treat Dancelab as a sub-set of Sprite Lab levels so have to check and set that before GamelabJr.
       // Artist levels have @level.game.app of "turtle" and Playlab levels have @level.game.app of "studio"
-      // so need to set those too
+      // so need to set those too.
+      // Java Lab levels use hidden levels rather than projects as their example, so we pass the URL directly.
```

Comment:
I get what you are saying here in general, but I think it would help to spell out the type of `example` here: a string representing a project key or a url.

Reply:
Fixed